### DISCUSSION TITLE
无人机用途为啥是灰色不可选状态

### DISCUSSION THREAD

**Discussion by sur42tur** [MaaAssistantArknights/MaaAssistantArknights#4876] [Q&A] [answered]
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/115613669/b5362dcb-3568-4243-99b5-9d7092d0247d)

**Comment by ABA2396** [comment, marked as answer]
因为你开了自定义基建配置，走的是配置文件，界面里无人机的选项是无效的
